scripts/verify-install.sh

```diff
@@ -132,20 +137,20 @@ for signature in $SIGNATURES; do
   echo ""
 
   # Then make sure that the hash of the installed binaries can be found in the
-  # signed list of hashes.
-  if ! grep -q "$LND_SUM" "$signature"; then
-    echo "ERROR: Hash $LND_SUM for lnd not found in $signature: "
+  # manifest.
+  if ! grep -q "$LND_SUM" "$MANIFEST"; then
+    echo "ERROR: Hash $LND_SUM for lnd not found in $MANIFEST: "
     cat "$signature"
     exit 1
   fi
 
-  if ! grep -q "$LNCLI_SUM" "$signature"; then
```

Comment:
interesting, so we could actually be signing different manifests, but the signatures would verify anyway?

Reply:
We used the `--clear-sign` flag before which overwrote the `--detach-sign` flag. So we didn't actually produce detached signatures.

If we start creating detached signatures, then the manifest they signed must be specified when verifying. So the manifest must match exactly and be present on the file system. The `.sig` file will only contain the raw signature, nothing more.
That's why we now have to look in the manifest text file for the hash.

I didn't do this at first because the manifest text file wasn't yet fully reproducible (because of the vendor mismatch and the sorting of the file itself). But now that that problem is out of the way we can actually start all signing the same manifest file and just upload the detached signature.